cypress/integration/utils/credentials-test/enter-credentials-cognito-s3.js

```diff
@@ -1,57 +1,99 @@
+require("cypress-xpath")
 const enterCredentialsCognitoS3 = () => {
-  var credentials = Cypress.env()
   cy.get('input[placeholder="XX-XXXX-X:XXXXXXXX-XXXX-1234-abcd-1234567890ab"]')
     .focus()
-    .type(credentials.AWS_IDENTITY_POOL_ID)
+    .type(Cypress.env().AWS_IDENTITY_POOL_ID)
   cy.get('input[placeholder="XX-XXXX-X"]')
     .first()
     .focus()
-    .type(credentials.AWS_AUTH_REGION)
+    .type(Cypress.env().AWS_AUTH_REGION)
   cy.get('input[placeholder="XX-XXXX-X_12ab34cd9"]')
     .focus()
-    .type(credentials.AWS_USER_POOL_ID)
+    .type(Cypress.env().AWS_USER_POOL_ID)
   cy.get('input[placeholder="26-char alphanumeric string"]')
     .focus()
-    .type(credentials.AWS_USER_POOL_WEB_CLIENT_ID)
+    .type(Cypress.env().AWS_USER_POOL_WEB_CLIENT_ID)
   cy.get('input[placeholder="Name of the bucket"]')
     .focus()
-    .type(credentials.AWS_STORAGE_BUCKET)
+    .type(Cypress.env().AWS_STORAGE_BUCKET)
   cy.get('input[placeholder="XX-XXXX-X"]')
     .last()
     .focus()
-    .type(credentials.AWS_STORAGE_REGION)
+    .type(Cypress.env().AWS_STORAGE_REGION)
   cy.get('input[class="MuiInputBase-input MuiInput-input"]')
     .last()
     .focus()
-    .type(credentials.COGNITO_USER_PASS_LENGTH)
-  if (credentials.COGNITO_USER_PASS_REQUIRE_LOWERCASE == "TRUE") {
-    cy.get('input[class="PrivateSwitchBase-input-65"]').eq(0).click()
+    .type(Cypress.env().COGNITO_USER_PASS_LENGTH)
+  if (Cypress.env().COGNITO_USER_PASS_REQUIRE_LOWERCASE === "TRUE") {
+    cy.xpath(
+      "./html/body/div[3]/div[3]/div/div[2]/div/div[2]/div[8]/div[3]/div/button[1]/span[1]/span/span[1]/input"
```

Comment:
use ids instead of long paths with generic elements.

paths that don't use ids are flaky and will break often.

Reply:
It depend on a foreign dependencie who's not able to at htis moment but i think its one of your project. So Ishould be able to do the change.

Reply:
Will be solve on credentials test